march_shared_classes/src/march_shared_classes/gait/subgait.py

```diff
@@ -247,7 +273,10 @@ def interpolate_subgaits(cls, base_subgait, other_subgait, parameter):
             base_subgait.version, other_subgait.version, parameter)
 
         duration = base_subgait.duration * parameter + (1 - parameter) * other_subgait.duration
-        return Subgait(joints, duration, base_subgait.gait_name, base_subgait.subgait_name, 'interpolated subgait',
+        gait_type = base_subgait.gait_type if parameter <= 0.5 else other_subgait.gait_type
```

Comment:
So if the parameter is 0.0 and the base_subgait is not used then it should the gait type of the base gait? It sounds like that should be the other way around
```suggestion
        gait_type = base_subgait.gait_type if parameter > 0.5 else other_subgait.gait_type
```

Reply:
parameter equaling zero means the base subgait, parameter equaling one means the other subgait. With `parameter==0` the base_subgait is returned.